models/sql_models.yaml

```diff
@@ -59,52 +61,32 @@ models:
         - select: "user_id"
           type: user_id
           entity: user
+          to_default_stitcher: true
         - select: "anonymous_id"
           type: anonymous_id
           entity: user
+          to_default_stitcher: true
   - name: rsOrderCreatedOrderCancelled
     model_type: sql_template
     model_spec:
       validity_time: 24h # 1 day
       materialization:
         output_type: ephemeral
         run_type: discrete
-      single_sql: "{% with OrderCreated = this.DeRef(\"inputs/rsOrderCreated\") OrderCancelled = this.DeRef(\"inputs/rsOrderCancelled\") OrderUpdated = this.DeRef(\"inputs/rsOrderUpdated\")%}
-         select
-             a.ANONYMOUS_ID,
-             a.user_id,
-             a.total_price_usd,
-             a.products,
-             a.payment_details_credit_card_company,
-             a.order_number,
-             a.timestamp,
-             a.cart_token,
-             a.financial_status,
-             b.anonymous_id as anonymous_id_order_cancelled,
-             b.user_id as user_id_order_cancelled ,
-             b.total_price_usd as total_price_usd_order_cancelled,
-             b.order_number as order_number_order_cancelled,
-             b.financial_status as financial_status_order_cancelled,
-             b.cart_token as cart_token_order_cancelled,
-             b.timestamp as timestamp_order_cancelled,
-             b.products as products_order_cancelled,
-             c.fulfillment_status
-         from {{OrderCreated}} a 
-         left join {{OrderCancelled}} b on a.user_id = b.user_id and a.order_number = b.order_number
-         left join {{OrderUpdated}} c on a.user_id = c.user_id and a.order_number = c.order_number
-          
-                {% endwith %}
-         "
+      single_sql: "{% with OrderCreated = this.DeRef(\"inputs/rsOrderCreated\") OrderCancelled = this.DeRef(\"inputs/rsOrderCancelled\") OrderUpdated = this.DeRef(\"inputs/rsOrderUpdated\")%} select a.ANONYMOUS_ID, a.user_id, a.total_price_usd, a.products, a.payment_details_credit_card_company, a.order_number, a.timestamp, a.cart_token, a.financial_status, b.anonymous_id as anonymous_id_order_cancelled, b.user_id as user_id_order_cancelled , b.total_price_usd as total_price_usd_order_cancelled, b.order_number as order_number_order_cancelled, b.financial_status as financial_status_order_cancelled, b.cart_token as cart_token_order_cancelled, b.timestamp as timestamp_order_cancelled, b.products as products_order_cancelled, c.fulfillment_status from {{OrderCreated}} a left join {{OrderCancelled}} b on a.user_id = b.user_id and a.order_number = b.order_number left join {{OrderUpdated}} c on a.user_id = c.user_id and a.order_number = c.order_number\n{% endwith %} "
```

Comment:
@yajuaryarudderstack can you validate this as well.

Reply:
Yes @nidhilashkari17 my bad, I executed `run` command and added Terminal stack here, forgot pushing the code. Have done that now.